cli/api/controller_client.go

```diff
@@ -0,0 +1,116 @@
+package api
+
+import (
+	"fmt"
+	api "github.com/amalgam8/amalgam8/cli/client"
+	"github.com/amalgam8/amalgam8/cli/common"
+	"github.com/amalgam8/amalgam8/cli/utils"
+	"github.com/urfave/cli"
+	"io"
+	"net/http"
+)
+
+type controller struct {
+	ctx    *cli.Context
+	client api.Client
+	debug  bool
+}
+
+// ControllerClient .
+type ControllerClient interface {
```

Comment:
This custom client is going to be deleted once the official a8 ctrl client provides more functionality.

I don't think we need to make changes to this and related files (e.g., client/client.go) because we are not going to include them in the final version of the CLI.